app/src/main/java/be/chvp/nanoledger/ui/common/TransactionFormComponents.kt

```diff
@@ -141,8 +142,12 @@ fun TransactionForm(
             }
             val postings by viewModel.postings.observeAsState()
             postings?.forEachIndexed { i, posting ->
-                val showAmountHint = posting.first == "" && posting.third == ""
-                PostingRow(i, posting, showAmountHint, viewModel)
+                val isNote = posting.first == "" && posting.third == "" && posting.fourth != ""
+                // do not show notes rows in the UI
+                if (!isNote) {
```

Comment:
If we make `Quadruple` a more bespoke class, we can probably move this to a helper function in that class.